### ISSUE TITLE
request: win32 binary

### ISSUE THREAD

**Issue by steom** [3gstudent/Eventlogedit-Evolution#13]
is it possible to have binaries for windows 32bit?

**Comment by 3gstudent** [comment]
Of course, you don't need to consider the difference in memory. You just need to compile it into 32 bits

**Comment by steom** [comment]
I meant if you are going to release binaries also for 32bit 
In https://github.com/3gstudent/Eventlogedit-evtx--Evolution/releases three are only 64bit binaries

**Comment by 3gstudent** [comment]
I won't think about it for the moment. It's just a program for testing, and it's not weaponized enough
